tests/test_html_output.py

```diff
@@ -25,7 +25,27 @@ def browser():
 
 def test_success(docker_main, browser):
     docker_main.run(config, additional_parameters="--html-log")
+    check_html_log(docker_main.artifact_dir, browser)
+
+
+def test_success_clean_build(docker_main, browser):
+    docker_main.run(config, additional_parameters="--html-log --clean-build")
+    check_html_log(docker_main.artifact_dir, browser)
+
+
+def test_success_nonci(docker_nonci, browser):
```

Comment:
You can combine this test and 'test_success' by using docker_main_and_nonci fixture

Reply:
I don't want to. Different "main" code is executed, so both of these cases should be checked.
And I prefer to follow "one test - one assert" rule (not literally =) ).